Gemfile

```diff
@@ -4,5 +4,8 @@ group :test do
   gem 'rake'
   gem 'rspec'
   gem 'beaker', '~>1.17.0'
+  if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
```

Comment:
That doesn't mean that it's a bad idea to make your code as explicit as possible, so that the next person who reads it knows exactly what it's trying to do :)